### PR TITLE
docs(portal): enhance documentation for onboarding process

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* enhance portal description to provide a script to skip the clearinghouse step for a successful onboarding

## Why

Without the script the clearinghouse step will fail, since there isn't a clearinghouse FOSS application

## Issue

Refs: #115

## Pre-review checks

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: charts/umbrella/README.md [hunk 1]
```diff
@@ -330,7 +338,28 @@ TBD.
...
+    AND ac.application_checklist_entry_type_id = 6
+    AND (ac.application_checklist_entry_status_id = 4 OR ac.application_checklist_entry_status_id = 1)
+),
+updated AS (
+ UPDATE portal.application_checklist
+     SET application_checklist_entry_status_id = 3
+     WHERE application_id IN (SELECT Id FROM applications)
+     RETURNING *
+)
+INSERT INTO process_steps (id, process_step_type_id, process_step_status_id, date_created, date_last_changed, process_id, message)
```
Comment (gomezbc) [inline charts/umbrella/README.md:359]: The schema needs to be set, or else it won't find the table. (portal.process_steps)

File: charts/umbrella/README.md [hunk 2]
```diff
@@ -330,7 +338,28 @@ TBD.
 
 #### Get to know the Portal
 
-Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file).
+Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file). Since the onboarding process requires the clearinghouse to work properly, but clearinghouse currently isn't available as a FOSS application you can skip the step with the following SQL Script which must be executed against the portal database.
```
Comment (evegufy) [inline charts/umbrella/README.md:341]: could you please add the whole information after the "Proceed with the login..." part?
In a section that might called something like "Note for onboarding process"

Clearinghouse should also b camelcase and the link [here](https://github.com/eclipse-tractusx/portal-assets/blob/v2.1.0/docs/developer/Technical%20Documentation/Interface%20Contracts/Clearinghouse.md) might also be useful.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/umbrella/README.md [hunk 3]
```diff
@@ -330,7 +338,28 @@ TBD.
...
+
+```sql
+WITH applications AS (
+    SELECT distinct ca.id as Id, ca.checklist_process_id as ChecklistId
+    FROM portal.company_applications as ca
+             JOIN portal.application_checklist as ac ON ca.id = ac.application_id
+    WHERE 
+      ca.application_status_id = 7 
+    AND ac.application_checklist_entry_type_id = 6
+    AND (ac.application_checklist_entry_status_id = 4 OR ac.application_checklist_entry_status_id = 1)
```
Comment (gomezbc) [inline charts/umbrella/README.md:351]: Currently, the clearinghouseTokenAddress default value is set to "http://keycloak.tx.test/realms/example/protocol/openid-connect/token". When the process worker makes a request to Keycloak with this default address, the request fails. Despite the failure, the status_id remains set to 3.

We could change `ac.application_checklist_entry_status_id = 4` condition to `ac.application_checklist_entry_status_id = 3`, change the default clearinghouseTokenAddress to centralidp to make it fail or explain it.

I'm not sure which could be the best approach

Comment (gomezbc) [reply charts/umbrella/README.md:351]: Screenshot 2024-08-07 at 17-38-03 Mondragon-X Portal

**Review by gomezbc — APPROVED** [review]

File: charts/umbrella/README.md (hunk 2 above)
Comment (Phil91) [reply charts/umbrella/README.md]: adjusted

File: charts/umbrella/README.md [hunk 4]
```diff
@@ -383,6 +392,31 @@ In case that you have TLS enabled (see [Self-signed TLS setup (Optional)](#self-
 - <https://portal-backend.tx.test>
 - <https://portal.tx.test>
 
+#### Note for onboarding process
```
Comment (evegufy) [inline charts/umbrella/README.md:395]: should be a subsection of Get to know the Portal, so #####

File: charts/umbrella/README.md [hunk 5]
```diff
@@ -1,13 +1,22 @@
...
+      - [Use released chart](#use-released-chart)
+        - [Option 1](#option-1)
+        - [Option 2](#option-2)
+      - [Use local repository](#use-local-repository)
+        - [Option 1](#option-1-1)
+        - [Option 2](#option-2-1)
     - [E2E Adopter Journeys](#e2e-adopter-journeys)
       - [Data exchange](#data-exchange)
       - [Get to know the Portal](#get-to-know-the-portal)
+      - [Note for onboarding process](#note-for-onboarding-process)
```
Comment (evegufy) [inline charts/umbrella/README.md:19]: should be a subsection of Get to know the Portal

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/umbrella/README.md (hunk 5 above)
Comment (Phil91) [reply charts/umbrella/README.md]: done

**Pull request merged**